### PR TITLE
Improvement to enable/disable ckpt in WaitAny.

### PR DESCRIPTION
**Pull request by karya0**

### CODE REVIEW

File: mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp [hunk 1]
```diff
@@ -289,6 +289,7 @@ USER_DEFINED_WRAPPER(int, Waitany, (int) count,
         *local_index = i;
         return retval;
       }
+      DMTCP_PLUGIN_ENABLE_CKPT();
```
Comment (JainTwinkle) [inline mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp:292]: There are multiple returns. I'm afraid that the function can return before enabling CKPT.

Comment (xuyao0127) [reply mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp:292]: Yes, I have the same concern here.

Comment (JainTwinkle) [reply mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp:292]: We can use break instead of return.

Comment (karya0) [reply mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp:292]: Good catch. Fixed.

**Review by JainTwinkle — APPROVED** [review]

**Pull request merged**